### PR TITLE
Header/Landing Page

### PR DESCRIPTION
**Pull request by Robbie08**
Incorporated a Most Viable Product for the header/landing page. Included images of the beer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,12 @@
     "webpack-dev-server": "^3.4.1"
   },
   "dependencies": {
+    "bootstrap": "^4.3.1",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-icons": "^3.7.0",
+    "react-router": "^5.0.1",
```
Comment (jcan1995) [inline package.json]: Maybe don't include react-router related stuff just yet

File: package.json [hunk 2]
```diff
@@ -23,7 +23,12 @@
...
+    "react-router-dom": "^5.0.1",
+    "styled-components": "^4.3.2"
```
Comment (jcan1995) [inline package.json]: Not using styled components atm.

File: src/components/App.js [hunk 3]
```diff
@@ -1,11 +1,35 @@
 import React, { Component } from 'react';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (jcan1995) [inline src/components/App.js]: Take out bootstrap. Remove from dependencies as well.

File: src/components/App.js [hunk 4]
```diff
@@ -1,11 +1,35 @@
...
+import { FaHome } from 'react-icons/fa';
+import Header from './Header';
+
+/* imports from our pages */
+import About from '../pages/AboutPage';
```
Comment (jcan1995) [inline src/components/App.js]: Remove all unused imports

File: src/components/Header.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {FaCartArrowDown} from 'react-icons/fa';
+
+const Header = (status) => {
+
+    console.log(status); /* This value should return the authentication status */
```
Comment (jcan1995) [inline src/components/Header.js]: No console log.

File: src/pages/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+const HomePage = (prop) => {
+
+    console.log(prop); // to see what we are recieving form the prop
```
Comment (jcan1995) [inline src/pages/HomePage.js]: No console log

File: src/pages/AboutPage.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
Comment (jcan1995) [inline src/pages/AboutPage.js]: Let's get rid of all unused components.

File: src/pages/HomePage.js [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+            <h1>Welcome, Sign up to recieve 20% Off todays purchase!</h1>
+        </div>
+    )
+}
+
+/* This will display the home page for unauthenticated users:
+    Requirements: 
+        - User should be prompted to invite friends to recieve discount
+*/
+const DisplayAuth = (data) => {
```
Comment (jcan1995) [inline src/pages/HomePage.js]: Let's do some es6 de-structuring on all object references.

File: package.json (hunk 1 above)
Comment (Robbie08) [reply package.json]: Done!

**Pull request merged**